### PR TITLE
Add hotfix delete and refactored hotfix

### PR DESCRIPTION
**Pull request by romakot321**

### CODE REVIEW

File: time_api/api/lessons.py
Comment (prostoLavr) [inline time_api/api/lessons.py:1]: Replace global const to enum. authenticate.monitor() raises exception when access level is lower then monitor, but next you use it for authenticate teachers.

Comment (prostoLavr) [inline time_api/api/lessons.py:1]: Teachers can remove lessons, but you use the monitor level.

File: time_api/migrations/versions/3256e9b567f5_lesson_hotfix.py
Comment (prostoLavr) [inline time_api/migrations/versions/3256e9b567f5_lesson_hotfix.py:1]: Create new migration for a table schema that already is on server.

File: time_api/schemas/lessons.py
Comment (prostoLavr) [inline time_api/schemas/lessons.py:1]: Use sometype | None instead Optional[sometype]

Comment (prostoLavr) [inline time_api/schemas/lessons.py:1]: Don't use optional lesson_id in answer schema. Vue.JS will raise exception if ID is null.

File: time_api/services/lessons_hotfix.py
Comment (prostoLavr) [inline time_api/services/lessons_hotfix.py:1]: Catch Exception on commit creating and deleting lessons. 45-47 stings are not readable. 56 string is so long, use partial execution.

**Review by prostoLavr — CHANGES_REQUESTED** [review]

File: time_api/api/lessons.py
Comment (romakot321) [reply time_api/api/lessons.py:1]: Please, be more attentive. Before checking the access_level there is verification for the cancel of the lessons for the whole day. It was done to ensure that only teachers could cancel the lessons for the whole day, and the monitor(class representative) could cancel only a few lessons.

Comment (romakot321) [reply time_api/api/lessons.py:1]: Monitor can create hotfix/remove lessons.

**Review by romakot321 — COMMENTED** [review]
The required changes were made in the following commits

File: time_api/api/lessons.py
Comment (prostoLavr) [inline time_api/api/lessons.py:1]: Rename monitor to class_representive. Add to hotifx params required school_id when lesson_id is None. Think about access level.

File: time_api/schemas/lessons.py
Comment (prostoLavr) [reply time_api/schemas/lessons.py:1]: Use it, please

File: time_api/services/auth.py
Comment (prostoLavr) [inline time_api/services/auth.py:1]: Rename monitor to class_representative

Comment (prostoLavr) [inline time_api/services/auth.py:1]: Remove test data from production build

File: time_api/services/lessons.py
Comment (prostoLavr) [inline time_api/services/lessons.py:1]: Replace sigle condition on 77 line to lesson cycle with condition everything.

Comment (prostoLavr) [inline time_api/services/lessons.py:1]: Replace enumarate to range

Comment (prostoLavr) [inline time_api/services/lessons.py:1]: To start new line in condition use:
`(some == some and \n some == some)`

**Review by prostoLavr — CHANGES_REQUESTED** [review]

**Pull request merged**